diff --git a/botocore/awsrequest.py b/botocore/awsrequest.py
index a667bb5..828fe32 100644
--- a/botocore/awsrequest.py
+++ b/botocore/awsrequest.py
@@ -118,7 +118,7 @@ class AWSHTTPConnection(HTTPConnection):
             if line in (b'\r\n', b'\n', b''):
                 break

-    def _send_request(self, method, url, body, headers):
+    def _send_request(self, method, url, body, headers, *py36_extra):
         self._response_received = False
         if headers.get('Expect', b'') == b'100-continue':
             self._expect_header_set = True
@@ -126,7 +126,7 @@ class AWSHTTPConnection(HTTPConnection):
             self._expect_header_set = False
             self.response_class = self._original_response_cls
         rval = HTTPConnection._send_request(
-            self, method, url, body, headers)
+            self, method, url, body, headers, *py36_extra)
         self._expect_header_set = False
         return rval

@@ -143,7 +143,7 @@ class AWSHTTPConnection(HTTPConnection):
         msg = b"\r\n".join(bytes_buffer)
         return msg

-    def _send_output(self, message_body=None):
+    def _send_output(self, message_body=None, **py36_extra):
         self._buffer.extend((b"", b""))
         msg = self._convert_to_bytes(self._buffer)
         del self._buffer[:]
